WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Customs/WooShippingCustoms.swift

```diff
@@ -0,0 +1,69 @@
+import SwiftUI
+import WooFoundation
+
+struct WooShippingCustoms: View {
+    let informationIsCompleted: Bool
+    @ScaledMetric private var scale: CGFloat = 1.0
+
+    var body: some View {
+        HStack {
+            Text(Localization.customsTitle)
+                .font(.system(size: Layout.customsTitleFontSize, weight: .medium))
+                .foregroundColor(.primary)
+
+            Spacer()
+
+            Text(informationIsCompleted ? Localization.completedStatus : Localization.missingInfoStatus)
+                .font(.system(size: Layout.statusBadgeFontSize, weight: .medium))
+                .foregroundColor(.black)
+                .padding(.horizontal, Layout.statusBadgeHorizontalPadding)
+                .padding(.vertical, Layout.statusBadgeVerticalPadding)
+                .background(
+                    RoundedRectangle(cornerRadius: Layout.statusBadgeCornerRadius)
+                        .fill(informationIsCompleted ?
+                              Color.withColorStudio(name: .green, shade: .shade5) :
+                                Color.withColorStudio(name: .red, shade: .shade10))
+                )
+                .padding(.horizontal, 10)
+
+            Button {
+            } label: {
+                Image(systemName: "pencil")
+                    .resizable()
+                    .frame(width: Layout.pencilButtonSizeDimensions * scale,
+                           height: Layout.pencilButtonSizeDimensions * scale)
+            }
+            .tint(Color(.primary))
+        }
```

Comment:
Just FYI that we have a `PencilEditButton` component, if you wanted to use it here. (It would probably need to have support for custom dimensions added to it, but that could make the component more useful elsewhere too.)

Reply:
Awesome, I changed it here https://github.com/woocommerce/woocommerce-ios/pull/14729/commits/f9949fb184262a8048ac3b33b3c79c6243214b9f